gtsam/hybrid/tests/testHybridEstimation.cpp

```diff
@@ -432,6 +432,69 @@ TEST(HybridEstimation, ProbabilityMultifrontal) {
   EXPECT(assert_equal(discrete_seq, hybrid_values.discrete()));
 }
 
+/****************************************************************************/
+/**
+ * Test for correctness via sampling.
+ *
+ * Given the conditional P(x0, m0, x1| z0, z1)
+ * with meaasurements z0, z1, we:
```

Comment:
Spelling. (install vs code spell-checking) and maybe insert comments inline below

Reply:
Oh good catch. I've already added comments inline with the corresponding bullet numbers.